src/webui/www/public/statistics.html

```diff
@@ -1,4 +1,4 @@
-<h3>User Statistics</h3>
+<h3>QBT_TR(User Statistics)QBT_TR[CONTEXT=StatsDialog]</h3>
```

Comment:
Can you someone figure out why this doesn't work? I set it to greek locale ("Ελληνικά") but the string comes back untranslated even though there is a translation for it in the el.ts file.